### DISCUSSION TITLE
SMA Home Manager 1.0 kompatibel?

### DISCUSSION THREAD

**Discussion by CHB68** [camueller/SmartApplianceEnabler#334] [Installation und Betrieb]
ich habe bisher erfolglos versucht, den SMA Home Manger 1.0 einzubinden.
Kann jemand bestätigen, dass sae nur mit dem Home Manager 2.0 läuft?

**Comment by camueller** [comment]
Ich habe selbst den SAE bis vor einem Jahr mit einem SHM 1.0 betrieben. SAE-seitig gibt es keinen Grund, warum das nicht funktionieren sollte.

**Reply by CHB68** [reply]
Danke für die Rückmeldung. Dann werde ich mich weiter auf die Fehlersuche begeben.

**Reply by CHB68** [reply]
SAE läuft jetzt mit meinem alten SHM 1.0. 
Ich musste die semp.gateway.address in der Datei /etc/default/smartapplianceenabler auf die IP Adresse meines Raspberry setzten.
Danke für das tolle Tool!

**Reply by camueller** [reply]
Schön, dass es läuft. Ich habe festgestellt, dass die Dokumentation bzw. die Links nicht ganz aktuell waren - das habe ich gerade korrigiert:
https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/SEMP_DE.md
